packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -139,7 +139,7 @@ export function TransferPanel() {
     useAppContextActions()
   const { addPendingTransaction } = useTransactionHistory(walletAddress)
 
-  const { isArbitrumNova } = isNetwork(childChain.id)
+  const { isArbitrumOne } = isNetwork(childChain.id)
```

Comment:
We triggered CCTP flow whenever child chain is not Arbitrum Nova before. That means it was a false positive for Orbit chains.